libraries/ms-common/src/stm32f0xx/interrupt.c

```diff
@@ -0,0 +1,200 @@
+#include "interrupt.h"
+
+#include <stdint.h>
+#include <stdlib.h>
+
+#include "gpio.h"
+#include "gpio_cfg.h"
+#include "status.h"
+#include "stm32f0xx_exti.h"
+#include "stm32f0xx_misc.h"
+#include "stm32f0xx_rcc.h"
+#include "stm32f0xx_syscfg.h"
+
+#define NUM_INTERRUPT_GPIO_CHANNELS 3
```

Comment:
Is this defined somewhere in the STM library or is there some way of calculating it (compile-time)?